data/abilities/command-and-control/356d1722-7784-40c4-822b-0cf864b0b36d.yml

```diff
@@ -72,8 +72,8 @@
           $name=$wc.ResponseHeaders["Content-Disposition"].Substring($wc.ResponseHeaders["Content-Disposition"].IndexOf("filename=")+9).Replace("`"","");
           Get-Process | ? {$_.Path -like "C:\Users\Public\$name.exe"} | stop-process -f -ea $ErrAction;
```

Comment:
Does this need to be replaced as well?

Reply:
so changing this breaks it. not sure exaclty why but what will essentially happen as is if splunkd.exe is running when I'm trying to run an agent called test.exe, splunkd.exe will get stopped and test.exe will be downloaded and executed.

Reply:
So line 72 where `$name` is originally declared won't be relevant anymore (since we're no longer using that variable). The process should only get stopped if its name matches the name of the new sandcat executable that we're downloading and starting. We need to stop the process in order to overwrite the executable anyway (if it already exists)

Reply:
Okay I went back and tested this change and it works with getting rid of $name completely and using #{agents.implant_name} instead